docs/installing-unsupported-php-versions.md

```diff
@@ -0,0 +1,45 @@
+# If you ever have to use a (default) unsupported PHP version, proceed as follows.
```

Comment:
Also change this title when changing the filename and entry reference. 
A description below the title would make sure the why and how you would want to use this.
Something along the way of:
```
If you need to use a different PHP version other than the out-of-the-box supported versions than follow instructions below.
```